types/setting.go

```diff
@@ -1590,16 +1556,25 @@ func isValidChoice(choices []string, value string) bool {
 	return len(choices) == 0
 }
 
-func GetCustomizedDefaultSettings(defaultSettingCM *corev1.ConfigMap) (defaultSettings map[string]string, err error) {
+func GetCustomizedDefaultSettings(defaultSettingCM *corev1.ConfigMap) (defaultSettings, defaultBackupTargetSettings map[string]string, err error) {
 	defaultSettingYAMLData := []byte(defaultSettingCM.Data[DefaultSettingYAMLFileName])
 
+	defaultBackupTargetSettings = map[string]string{}
+
 	defaultSettings, err = getDefaultSettingFromYAML(defaultSettingYAMLData)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
 	// won't accept partially valid result
 	for name, value := range defaultSettings {
+		if name == string(SettingNameBackupTarget) ||
+			name == string(SettingNameBackupTargetCredentialSecret) ||
```

Comment:
nit: Some util libs can be considered for this kind of general operation. 
https://github.com/samber/lo